### PR TITLE
ctrl-f public service announcement. Towards #186.

### PR DESCRIPTION
**Pull request by mccalluc**
@afred?
screen shot 2016-02-22 at 7 39 15 pm

### CODE REVIEW

**Comment by caseyedavis12** [comment]
@mccalluc: Looks great. Is that the symbol for command on a PC?

**Comment by mccalluc** [comment]
@caseyedavis12 : on PC, it's just "control": look at the source to see the messaging.

File: app/views/layouts/transcript.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <script>
+          (function() {
+            // Anonymous function for variable scoping.
+            var message;
+            if (navigator.platform.indexOf("Mac")!==-1) {
+              message = '&#x2318; F';
+            } else if (navigator.userAgent.match(/iPod|iPad|iPhone/i)) {
+              message = 'the address bar at the top of the screen';
+            } else { 
+              message = 'control-F';
```
Comment (afred) [inline app/views/layouts/transcript.html.erb]: @mccalluc `control-F` is not accurate for Chrome on Android

Comment (afred) [inline app/views/layouts/transcript.html.erb]: Won't block on this though. Merging.

**Pull request merged**